src/main/java/com/aws/greengrass/componentmanager/ClientConfigurationUtils.java

```diff
@@ -91,7 +91,13 @@ private static void configureClientMutualTLS(ApacheHttpClient.Builder httpBuilde
         try {
             List<X509Certificate> trustCertificates = EncryptionUtils.loadX509Certificates(Paths.get(rootCAPath));
 
+#if ANDROID
+            // NOTE: android does not support "JKS"
+            KeyStore tmKeyStore = KeyStore.getInstance(KeyStore.getDefaultType());
+#else
             KeyStore tmKeyStore = KeyStore.getInstance("JKS");
+#endif
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;Invalid Character&#34;, @ expected

This file contains syntax errors and can&#39;t be analyzed by all of CodeGuru Reviewer&#39;s detectors. Fix the syntax errors and run the repository analysis or submit the pull request again.